src/components/RegisterFormCompany/RegisterFormCompany.tsx

```diff
@@ -0,0 +1,288 @@
+import React, { useMemo, useState } from 'react';
+import { TextField, InputAdornment, Box, Button, LinearProgress, Divider, MenuItem } from '@material-ui/core';
+import { AccountCircle, Lock, Email, Phone, Language, Room } from '@material-ui/icons';
+import { theme } from '../../utils/theme';
+import useAuthActions from '../../Auth/useAuthActions';
+import useTexts from '../../hooks/useTexts';
+import { Alert } from '@material-ui/lab';
+import { useForm } from 'react-hook-form';
+import config from '../../config/config';
+import useResponsive from '../../hooks/useResponsive';
+import { countryList } from '../CountryList/countryList';
+
+const EMAIL_VALIDATION_REGEX = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+interface Inputs {
+  email: string;
+  username: string;
+  password: string;
+  confirmPassword: string;
+  companyname: string;
+  number: string;
+  location: string;
+  website: string;
+}
+
+interface Props {
+  onSuccess?: () => void;
+}
+
+const RegisterFormCompany: React.FC<Props> = ({ onSuccess }) => {
+  const countrylist = countryList.map((country) => (
+    <MenuItem key={country} value={country}>
+      {country}
+    </MenuItem>
+  ));
+  const texts = useTexts();
+  const { register: userRegister } = useAuthActions();
```

Comment:
This registers a user and not a company, so this will not work currently (idk how company registration will go on the backend but i guess we need that for functionality)